pymeasure/instruments/thorlabs/thorlabspro8000.py

```diff
@@ -11,7 +11,7 @@
 # furnished to do so, subject to the following conditions:
 #
 # The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
+# all copies or @@substantial portions of the Software.
```

Comment:
what's this? :D

Reply:
My F*** keyboard sometimes ...

Reply:
^.^ You can avoid that if you check the content of each commit, before you commit, in whatever tool you use.

Reply:
You're right ...

Reply:
can you remove it again please?

Reply:
I changed it.